### DISCUSSION TITLE
Kratos hangs after successful registration?

### DISCUSSION THREAD

**Discussion by cswaney** [ory/kratos#3454] [Q&A]
I'm trying to run Kratos on an air-gapped system.

While testing `quickstart.yml` I've discovered that the registration self-service flow _appears_ to fail. Upon inspection, I discovered that the user is in fact added to the database and can subsequently login. However, an error occurs after  `PostRegistrationPrePersistHooks`, which prevents Kratos from returning a response. According to the logs (see below), Kratos thinks that the new user already exists.

I don't have any problems running the quickstart on my macOS laptop.

Any ideas what is going on here?

```
kratos-kratos-1                      | time=2023-08-25T01:07:17Z level=info msg=started handling request...
kratos-kratos-1                      | time=2023-08-25T01:07:32Z level=debug msg=Running PostRegistrationPrePersistHooks...
kratos-kratos-1                      | time=2023-08-25T01:07:32Z level=info msg=Encountered self-service flow error... an account with the same identifier (email, phone, username, ...) exists already stack_trace:
...
kratos-kratos-1                      | time=2023-08-25T01:07:32Z level=info msg=completed handling request ...
```

**Comment by vinckr** [comment]
Hello @cswaney 

>I'm trying to run Kratos on an air-gapped system.

Very interesting use case! If you can share more about it, feel free to [book a short call](https://meetings-eu1.hubspot.com/vincent-kraus) with me sometime, I would be interested in learning more.

Not sure what is going on though. You deployed Kratos with a proper DB? Are you using the latest version?

**Reply by cswaney** [reply]
Thanks for your response! I scheduled a quick call.

I've been testing with SQLite only at this point.
